src/resolvers.ts

```diff
@@ -236,7 +237,7 @@ export async function getResolvers(
   }
 
   function resolveFile(specifier: string): string {
-    return relativePath(path, loaders.resolveFilePath(resolvePath(path, specifier)));
+    return relativePath(path, loaders.resolvePreviewFilePath(resolvePath(path, specifier)));
```

Comment:
This is also confusing, since this code runs during build… but the difference between a “preview file path” and a “build file path” is only the hash, and the hash ultimately isn’t used in build because file references are remapped through the `aliases` map. I need to come up with some better names here.